packages/test/test-end-to-end-tests/src/test/benchmark/DocumentCreator.ts

```diff
@@ -102,14 +103,26 @@ export function benchmarkAll<T extends IBenchmarkParameters>(title: string, obj:
 		};
 		benchmarkMemory(t);
 	} else {
+		const runMethod = obj.run.bind(obj);
 		const t1: BenchmarkArguments = {
 			title,
 			...obj,
-			benchmarkFnAsync: obj.run.bind(obj),
```

Comment:
Nit: What's obj? Test object?

Reply:
Is the object we dynamically create when we call the benchmarkall:
ex. new (class PerformanceTestWrapper implements IBenchmarkParameters {